src/System.Memory/pkg/System.Memory.pkgproj

```diff
@@ -5,8 +5,8 @@
     <ProjectReference Include="..\ref\System.Memory.csproj">
       <SupportedFramework>net45;netcore45;wp8;wpa81;netcoreapp1.0;$(AllXamarinFrameworks)</SupportedFramework>
     </ProjectReference>
-    <ProjectReference Include="..\src\System.Memory.csproj">
-      <SupportedFramework>net45;netcore45;wp8;wpa81;netcoreapp1.0;$(AllXamarinFrameworks)</SupportedFramework>
+    <ProjectReference Include="..\src\System.Memory.builds">
+      <SupportedFramework>net45;netcore45;wp8;wpa81;netcoreapp1.1;$(AllXamarinFrameworks)</SupportedFramework>
```

Comment:
I do not think it does anything; and other projects with similar structure to this one do not have it either: https://github.com/dotnet/corefx/blob/master/src/System.Buffers/pkg/System.Buffers.pkgproj#L8